.github/PULL_REQUEST_TEMPLATE/release_pull_request.md

```diff
@@ -28,11 +28,12 @@ Release for Gutenberg Mobile v1.XX.Y
 
 ## Release Submission Checklist
 
-- [ ] Release number was bumped
+- [ ] Release number was bumped in `package.json` and `gutenberg/packages/react-native-editor/package.json`
 - [ ] Aztec dependencies are pointing to a stable release
```

Comment:
It looks like the stable release check was dropped, was that intentional?

Reply:
The check is still there, noticed that the line is not red. I think the minus for the list is making it look deleted.

Reply:
🤦‍♂️ You're totally right!